services/src/main/java/org/keycloak/protocol/oidc/grants/device/endpoints/DeviceEndpoint.java

```diff
@@ -111,6 +111,11 @@ public Response handleDeviceRequest() {
         AuthorizationEndpointRequest request = AuthorizationEndpointRequestParserProcessor.parseRequest(event, session, client,
             httpRequest.getDecodedFormParameters());
 
+        if(request.getInvalidRequestMessage() != null) {
```

Comment:
It might be better to insert a blank between `if` and `(request.` like:
`if (request.`